pkg/handler/project/project.go

```diff
@@ -132,3 +133,110 @@ func (h *handler) UpdateProjectStatus(c *gin.Context) {
 	// 3. return project data
 	c.JSON(http.StatusOK, view.CreateResponse[any](view.ToUpdateProjectStatusResponse(project), nil, nil, nil))
 }
+
+// Create godoc
+// @Summary	Create new project
+// @Description	Create new project
+// @Tags Project
+// @Accept json
+// @Produce json
+// @Param Authorization header string true "jwt token"
+// @Param Body body CreateProjectInput true "body"
+// @Success 200 {object} view.CreateProjectData
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /projects [post]
+func (h *handler) Create(c *gin.Context) {
+	body := CreateProjectInput{}
+	if err := c.ShouldBind(&body); err != nil {
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, body))
+		return
+	}
+
+	// TODO: can we move this to middleware ?
+	l := h.logger.Fields(logger.Fields{
+		"handler": "project",
+		"method":  "Create",
+		"body":    body,
+	})
+
+	if err := body.Validate(); err != nil {
+		l.Error(err, "validate failed")
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, body))
+		return
+	}
+
+	var startDate *time.Time
+	var err error
+	if body.StartDate != "" {
+		startDate = new(time.Time)
+		*startDate, err = time.Parse("2006-01-02", body.StartDate)
+		if err != nil {
+			l.Error(err, "failed to parse startDate")
+			c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, body))
+			return
+		}
+	}
+	country, err := h.store.Country.One(body.CountryID)
+	if err != nil {
+		l.Error(err, "failed to get country")
+		c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, err, body))
+		return
+	}
+
+	project := &model.Project{
+		Name:      body.Name,
+		Country:   country.Name,
+		Type:      model.ProjectType(body.Type),
+		Status:    model.ProjectStatus(body.Status),
+		StartDate: startDate,
+	}
+	if err := h.store.Project.Create(project); err != nil {
+		l.Error(err, "failed to create project")
+		c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, err, nil))
+		return
+	}
+
+	// create project account manager
+	accountManagerID, err := model.UUIDFromString(body.AccountManagerID)
```

Comment:
as I remember the validator can check UUID is invalid or not. So I think we don't need to check it here.